### DISCUSSION TITLE
beatmap search listing: 3 columns of maps vs 2 columns (website)

### DISCUSSION THREAD

**Discussion by aplotor** [ppy/osu#29895] [Q&A] [answered]
in lazer client beatmap search listing, 3 columns of maps are displayed on my screen size while the website displays 2 columns of maps at the same screen size. it would be nice for the client to match the website in this regard (or have a settings option) as 3 columns requires a wider range of horizontal hand movement (when using a pen+tablet) to move across almost the entire screen to perform actions on the cards which gets pretty uncomfortable with repetition especially for people with wrist issues. i also feel like it takes longer to interpret the maps in a 3-column layout while scrolling through compared to a 2-column layout because your eyes need to scan further to both sides from the middle

thanks !

**Comment by bdach** [comment, marked as answer]
This was intentionally changed in the client to be able to display more maps on the screen and be more in line with the rest of the UI: https://github.com/ppy/osu/pull/28074
